lib/phoenix/controller.ex

```diff
@@ -1,5 +1,7 @@
 defmodule Phoenix.Controller do
   import Plug.Conn
+  import Plug.Conn.Query
```

Comment:
Please don't import `Plug.Conn.Query` to use a single function. Call Plug.Conn.Query.encode instead. Thank you.